ASUWebPlatforms/webspark-ci
# WS2-1623: Create MVP version of monorepo for ASU modules

### Description

As per our conversation a few days ago, I have gone with an MVP approach. In order to make it a truly viable product, I needed to do quite a bit of dependency cleanup. So, it turns out that in order to have the asu_brand module, we need 4 other modules. So, I have added all five to the MVP effort and am now ready for PR review. The included modules are:

- asu_brand
- asu_react_core
- asu_react_integration
- asu_config_utility (I split out this functionality from the webspark_utility module so that we have a tool just for config management)
- asu_user

QA Steps:
The Github Actions integration has been tested and has passed QA. But, to test whether the asu_brand module is able to be added to a non-Webspark site, you can do the following:
- Follow these directions to spin up a vanilla Drupal 10 site locally in DDEV: https://ddev.readthedocs.io/en/stable/users/quickstart/#drupal-drupal-10
- Edit the composer.json file to add "asuwebplatforms/asu_brand": "2.14.1" to the “require” section.
- Edit the "minimum-stability" key to be "dev" instead of "stable" and save the file.
- Run ddev composer update
- Launch the site and after initializing it, attempt to enable the asu_brand module by navigating to /admin/modules and clicking the checkbox next to the module, and then clicking “Install.” You should see a message that says You must install the ASU React Integration, ASU React Core, Paragraphs, Entity Reference Revisions, ASU User, CAS, External Authentication, ASU Config Utility, Configuration Update Base modules to install ASU Brand. Click "Continue."
- Navigate to /admin/structure/block and click the “Place block” button in the Header region.
- Click the “Place block” button on the row for “ASU brand header” and click “Save block”
- Navigate to the home page. Now you should see the ASU header on the page. If so, this passes.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1623)

### Checklist


### Verified in browsers 


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
